src/matchers.ts

```diff
@@ -326,11 +341,7 @@ export async function matchesLlmRubric(
 
   const rubricPrompt = grading?.rubricPrompt || DEFAULT_GRADING_PROMPT;
   invariant(typeof rubricPrompt === 'string', 'rubricPrompt must be a string');
-  const prompt = nunjucks.renderString(rubricPrompt, {
-    output: JSON.stringify(llmOutput).slice(1, -1),
-    rubric: JSON.stringify(rubric).slice(1, -1),
```

Comment:
Are we losing some of the JSON escaping by removing this? 
 e.g. added in https://github.com/promptfoo/promptfoo/commit/fdc578753b2b0e8ca67d0e51bdfc31f83294aac4#diff-b9eac0331d722656ef4dab1ad43886fb5e80547ce075a6eed0c2d64dbf64bb61R305

Reply:
it just got moved to here: https://github.com/promptfoo/promptfoo/pull/1583/files#diff-ba050ec0aef94e7c13a600f1aaaf3747c9624543ce666b25035dda59a6faf0c7R375-R376

Reply:
ah shit I'm sorry